models/articles.model.js

```diff
@@ -13,3 +15,34 @@ exports.fetchArticleById = async (id) => {
     throw err;
   }
 };
+exports.fetchArticles = async () => {
+  try {
+    let baseQuery = `SELECT
+    a.author,
+    a.title,
+    a.article_id,
+    a.topic,
+    a.created_at,
+    a.votes,
+    a.article_img_url,
+    count(c.comment_id) AS comment_count
+FROM
+    articles AS a
+    LEFT JOIN comments AS c ON c.article_id = a.article_id
+GROUP BY
```

Comment:
You only need to group by a unique column. Grouping by comment.article_id would suffice here :)

Reply:
This was really helpful!